FavoriteArtists/FavoriteArtists/Artist.m

```diff
@@ -0,0 +1,76 @@
+//
+//  Artist.m
+//  FavoriteArtists
+//
+//  Created by Bohdan Tkachenko on 11/30/20.
+//
+
+#import "Artist.h"
+
+@implementation Artist
+
+- (instancetype)initWithArtistName:(NSString *)aName
+                        formedYear:(int)aYear
+                         biography:(NSString *)aBiography
+{
+    if (self = [super init]) {
+        _artistName = aName.copy;
+        _formedYear = aYear;
+        _biography = aBiography.copy;
+    }
+    return self;
+}
+
+- (instancetype)initWithSearchResults:(NSDictionary *)dictionary
+{
+    NSArray *artists = [dictionary objectForKey:@"artists"];
+    
```

Comment:
```suggestion
     if (![artists isKindOfClass:[NSArray class]]) {
        return nil;
    }
```
So I noticed that if you search for an artist that doesn't exist in the api the app would crash, I saw you did a check for the dictionary being a dictionary class in your network call. The issue is the json still comes back as a dictionary just the value for artists is null. Adding the above suggestion does solve this problem, and I love how you added an alert for the user to know it did not find the result, so great work there.